mender-auth/cli/actions.cpp

```diff
@@ -61,39 +61,52 @@ shared_ptr<MenderKeyStore> keystore_from_config(
 	return make_shared<MenderKeyStore>(pem_file, ssl_engine, static_key, passphrase);
 }
 
-DaemonAction::DaemonAction(shared_ptr<MenderKeyStore> keystore) :
-	keystore_(keystore) {
-}
-
-ExpectedActionPtr DaemonAction::Create(const conf::MenderConfig &config, const string &passphrase) {
-	auto key_store = keystore_from_config(config, passphrase);
-
-	auto err = key_store->Load();
+error::Error do_boostrap(shared_ptr<MenderKeyStore> keystore, bool force) {
+	auto err = keystore->Load();
 	if (err != error::NoError && err != KeyStoreNoKeysError) {
-		return expected::unexpected(err);
+		return err;
 	}
-	if (err == KeyStoreNoKeysError) {
+	if (err == KeyStoreNoKeysError || force) {
 		log::Info("Generating new RSA key");
-		auto err = key_store->Generate();
+		auto err = keystore->Generate();
```

Comment:
I know it's on purpose, but maybe we should still think twice about shadoing `err` here?

Reply:
It is not on purpose. Thanks!